cfn-guard/README.md

```diff
@@ -89,6 +90,8 @@ The available operations are:
 
 * `==` - Equal
 * `!=` - Not Equal
+* `<=` - Smaller As
```

Comment:
I think it would be more familiar to use "Less Than" and "Greater Than" for these labels.

Reply:
The token also suggests that the evaluation would be "Less Than or Equal To" which is not how your rule evaluates. We might want to clear that up. :+1: